Yoast/wordpress-seo
# Only omit Twitter image if URL is same as OG image

## Context
* Filtering `wpseo_twitter_image_size` to use a different image size for Twitter, but the Twitter image meta tag is currently omitted. This change will allow the Twitter image to be a different size than the OG image.

## Summary
This PR can be summarized in the following changelog entry:

* Only omit Twitter image if URL is same as OG image

## Relevant technical choices:

* Checking the URL of the possible Twitter image against the URL of the OG image allows us to use only omit the Twitter image tag if it would be redundant with the OG image tag.

## Test instructions
This PR can be tested by following these steps:

* Add the filter below to `functions.php`
* Create a new post and set the featured image
* With the patch applied, the Twitter image tag should appear in addition to the OG image tag. Without the patch applied, only an OG image would be shown.

```

/**
 * Get social image size.
 *
 * @return string Social image size to use.
 */
function yoast_pr_get_social_image_size_twitter() {
	return 'large';
}
add_filter( 'wpseo_twitter_image_size', 'yoast_pr_get_social_image_size_twitter' );
```

## UI changes

## Documentation

## Quality assurance
